testinfra/modules/python_package.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+# Copyright © 2015 Philippe Pepiot
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import unicode_literals
+
+import re
+
+from testinfra.modules.base import Module
+
+
+class PythonPackage(Module):
+    """Test status and version of Python package."""
+
+    def __init__(self, pkg_name, pip_path='pip'):
```

Comment:
I would prefer a virtualenv parameter here instead of pip_path (and exec pip from `os.path.join(virtualenv, "bin", "pip")`)